lib/package/components/button.css

```diff
@@ -2,36 +2,131 @@
 
 .button {
   border: 0;
-  width: 10rem;
   padding: 0.75rem 1.25rem;
-  font-size: 1.15rem;
+  font-size: 1rem;
   cursor: pointer;
+  background: none;
+}
+
+.button-large {
+  width: 10rem;
+  padding: 0.75rem 1.25rem;
 }
+
 .button-primary {
   color: #fff;
   background-color: var(--bg-primary);
 }
-.button-primary-outline {
+
+.button-primary:hover {
   color: var(--bg-primary);
   background-color: transparent;
   border: 2px solid var(--bg-primary);
 }
-.button-round {
-  border-radius: 2rem;
+
+.button-success {
+  color: #fff;
+  background-color: var(--bg-success);
 }
-.button:hover {
-  transition: var(--transition);
+
+.button-info {
+  color: #fff;
+  background-color: var(--bg-info);
 }
-.button-primary:hover {
+
+.button-warning {
+  color: #fff;
+  background-color: var(--bg-warning);
+}
+
+.button-danger {
+  color: #fff;
+  background-color: var(--bg-danger);
+}
+
+.button-primary-outline {
   color: var(--bg-primary);
   background-color: transparent;
   border: 2px solid var(--bg-primary);
-  transform: translateY(-3px);
-  transition: background-color 1s ease, color 0.3s ease;
 }
+
 .button-primary-outline:hover {
   color: #fff;
   background-color: var(--bg-primary);
-  transform: translateY(-3px);
-  transition: background-color 1s ease, color 0.3s ease;
+}
+
+.button-success-outline {
+  color: var(--bg-success);
+  background-color: transparent;
+  border: 2px solid var(--bg-success);
+}
+
+.button-success-outline:hover {
+  color: #fff;
+  background-color: var(--bg-success);
+}
+
+.button-info-outline {
+  color: var(--bg-info);
+  background-color: transparent;
+  border: 2px solid var(--bg-info);
+}
+
+.button-info-outline:hover {
+  color: #fff;
+  background-color: var(--bg-info);
+}
+
+.button-warning-outline {
+  color: var(--bg-warning);
+  background-color: transparent;
+  border: 2px solid var(--bg-warning);
+}
+
+.button-warning-outline:hover {
+  color: #fff;
```

Comment:
don't directly use color code here. Make variable

Reply:
Thank You Kajal, made changes as you suggested.